### PR TITLE
Update stylelint scales to allow 3px border-radius

### PR DESCRIPTION
**Pull request by sixhours**
Related to #83361

## Proposed Changes

* Update stylelint scales to allow `border-radius` properties of `3px` in addition to 2 and 4.
* Remove multiple comments where we've ignored the `3px` value.

## Testing Instructions

* Visual inspection and passing tests should be sufficient.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/components/popover-menu/style.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	border-radius: 0;
 	cursor: pointer;
 	display: block;
-	font-size: $font-body-small;
+	font-size: 0.875rem;
```
Comment (markbiek) [inline client/components/popover-menu/style.scss:15]: Is this change intentional? It doesn't seem related to `border-radius`?

Comment (sixhours) [reply client/components/popover-menu/style.scss:15]: Yeah, that variable was causing a build failure (I suspect because as part of the ETK it doesn't have access to the Calypso font size variables) so I updated it to hard-code the value. I should have added that to the description, thanks for catching it!

It remains to be seen why it wasn't an issue until now. 🤔

**Review by markbiek — APPROVED** [review]
LGTM! I did a couple of different types of searches and didn't find any additional spots to change.
